docs/get-started/run.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_label: 'Run the transformation pipeline'
+sidebar_position: 6
+---
+
+# Run the transformation pipeline
+
+In the previous steps of this tutorial we spent some time to create necessary components and connect
+them all together to form our pipeline DAG. The first step before getting to a proper running pipeline
+is to make sure that the DAG we created id valid.
+
+To do so the sub-command `holium project export --no-write` is available:
+```shell
+$ holium project export --no-write
+current local project holds a healthy transformation pipelin
+```
+
+Now that we know that our pipeline DAG is sane we will focus on the execution.
+
+The only thing left to do before running our transformation pipeline is to define **portations** to
+import data to Holium and export data from Holium. In our case we will apply two import **portations**
+to our sources and one export **portation** to our shaper.
+
+The different files that we will use as our imported data can be found [in our dedicated repository](https://github.com/polyphene/getting-started/tree/main/assets/wasm).
+
+In the first file, `avocado.json` we can find all retails sales details for avocado in the US. The second
+file is the year we want to conduct our operation on.
+
+To create a new **portation** the `holium portation create` sub-command can be used. We will also review
+the created **portations** by using the sub-command `holium portation list` :
+```shell
+$ holium portation create --direction toHolium --node-type source --node-name avocado_sales --file-path avocado.json  --file-format json
+new object created: to:source:avocado_sales
+
+$ holium portation create --direction toHolium --node-type source --node-name selected_year --file-path selected_year.json  --file-format json
+new object created: to:source:selected_year
+
+$ holium portation create --direction fromHolium --node-type shaper --node-name sales_results --file-path sales_results.json  --file-format json
+new object created: from:shaper:sales_results
+```
+
+As we have now defined all necessary information to have a proper transformation pipeline DAG and data 
+sourcing we can execute it !
+
+To do so the sub-command `holium project run` is at your disposal:
+```shell
+$ holium project run
```

Comment:
I personally get `execution failed for transformation: avocado_operations` with provided assets.

Do not hesitate to ping me if/when needed to investigate.

Reply:
Are you using the latest hotfix on `holium` ? If not please try doing so. Otherwise I do not really know where it could come from as I just tested it on my side and it works.

Reply:
Thanks, solved it.